README.md

```diff
@@ -1,6 +1,7 @@
 # AGAVE : Advanced GPU Accelerated Volume Explorer
 
-AGAVE is a desktop application for viewing multichannel volume data. Several formats are supported, including OME-ZARR 0.4, OME-TIFF and Zeiss .czi files.
+AGAVE is a desktop application for viewing multichannel volume data. Several formats are supported, including OME-ZARR 0.4 and 0.5, OME-TIFF and Zeiss .czi files.
+![screenshot](https://github.com/user-attachments/assets/b96618f2-7020-4b93-936e-9b32b795ea83)
```

Comment:
```suggestion
AGAVE is a desktop application for viewing multichannel volume data. Several formats are supported, including OME-ZARR 0.4 and 0.5, OME-TIFF and Zeiss .czi files.

![screenshot](https://github.com/user-attachments/assets/b96618f2-7020-4b93-936e-9b32b795ea83)
```

Reply:
Tiny nit, I think markdown syntax advises keeping a line of spacing around images